exotic.py

```diff
@@ -1143,19 +1185,20 @@ def parse_args():
             print("\nYour current working directory is: ", os.getcwd())
             print("\nPotential initialization files I've found in " + os.getcwd() + " are: ")
             [print(i) for i in g.glob(os.getcwd() + "/*.txt")]
-            initfilename = str(input("\nPlease enter the Directory and Filename of your Initialization File: "))
-            if initfilename == 'ok':
-                initfilename = "/Users/rzellem/Documents/EXOTIC/inits.txt"
 
             # Parse input file
             while True:
                 try:
+                    initfilename = str(input("\nPlease enter the Directory and Filename of your Initialization File: "))
+                    if initfilename == 'ok':
+                        initfilename = "/Users/rzellem/Documents/EXOTIC/inits.txt"
                     with open(initfilename, 'r') as f:
                         inits = f.readlines()
-                        break
+                    break
                 except FileNotFoundError:
-                    print("Initialization file not found. Please try again.")
-                    initfilename = str(input("\nPlease enter the Directory and Filename of your Initialization File: "))
+                    print("Error: Initialization file not found. Please try again.")
+                except IsADirectoryError:
+                    print('Error: Entered a directory. Please try again.')
```

Comment:
Added an additional validation check to inform user they entered a directory instead of a file.